projects/devices/robotiq/protos/Robotiq3fGripper.proto

```diff
@@ -1,11 +1,11 @@
-#VRML_SIM R2020a utf8
+#VRML_SIM R2020b utf8
```

Comment:
```suggestion
#VRML_SIM R2020a utf8
```

Please do not update the header when targeting the master branch.

Reply:
Was on accident, will look out for it in the future